### DISCUSSION TITLE
Debug snaps

### DISCUSSION THREAD

**Discussion by Damantino** [MetaMask/snaps#1186] [Q&A] [answered]
Hello snap folks,

I've been toying with snaps and I'm not able to do some dead simple debugging.
I've read the documentation and have set up some console.logs and have opened `background.html` with no success.

To clarify I'm trying to tweak the same source code from the `getFees()`  example.
I would like to console.log the response coming from the fetch method, how could I do that?

Thank you and sorry if it's a very stupid question.

**Comment by ziad-saab** [comment]
Hi @Damantino ! Can you:

1. Share some of the code you're using
2. Show a screenshot of the console output from `background.html`

Thanks!

**Reply by Damantino** [reply]
Sure, here is the code I'm currently working with:


then((response) => {
        console.log('Response:', response);
        return wallet.request({
          method: 'snap_confirm',
          params: [
            {
              prompt: getMessage(origin),
              description:
                'This custom confirmation is just for display purposes.',
              textAreaContent: 'Current fee estimates', // here I would like to display response
            },
          ],
       });
      });`

And here is a screenshot of the output from `background.html`

![Screenshot from 2023-02-09 18-11-20](https://user-images.githubusercontent.com/50982542/217888138-0f27ccd3-df96-408b-a624-ef065e4f917d.png)

**Comment by ziad-saab** [comment, marked as answer]
I can see your `console.log`'s output in the screenshot you sent. Is this not what you're expecting?

**Comment by ziad-saab** [comment]
Actually it looks like the URL you're trying to fetch is wrong. Looks like there might be an extra `'` in the URL string?

**Reply by Damantino** [reply]
Certainly, was not expecting that response, I saw it now. Thank you very much!

**Comment by Damantino** [comment]
Given that was a mistake by me should we close this topic?

**Reply by Montoya** [reply]
Sure! Locking!
